docs/docsite/rst/dev_guide/testing_units_modules.rst

```diff
@@ -341,33 +352,32 @@ Running the main function
 If you do want to run the actual main function of a module you must import the module, set
 the arguments as above, set up the appropriate exit exception and then run the module::
 
-    # This test is based around pytest's features for individual test functions
```

Comment:
We should probably have at least one pytest based example in the document since they are quite common and reduce the barrier to entry for someone to write their first unit test.  I'm not sure about changing this over to a unit_test based example.